Chandra/Time/Time.py

```diff
@@ -509,6 +497,8 @@ def convert_vals(vals, format_in, format_out):
 
     :returns: converted values as either scalar or numpy array
     """
+    from . import _axTime3 as axTime3
```

Comment:
I replied in the wrong spot, but for lazy loads of external packages that is a good idea.